dirty_cat/_datetime_encoder.py

```diff
@@ -198,7 +198,7 @@ def transform(self, X, y=None) -> np.array:
 
         Returns
         -------
-        np.array, shape (n_samples, n_features_out_)
+        np.array, shape (n_samples, n_features_out\_)
```

Comment:
```suggestion
        np.array, shape (n_samples, n_features_out_)
```
Might be a mistake, this causes a warning in the current build:
```
 dirty_cat/_datetime_encoder.py:190
  /home/runner/work/dirty_cat/dirty_cat/dirty_cat/_datetime_encoder.py:190: DeprecationWarning: invalid escape sequence \_
```